database/config.py

```diff
@@ -1,9 +1,15 @@
+import os
+import urlparse
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
-from sqlalchemy.ext.declarative import declarative_base
 
 
-engine = create_engine('mysql+mysqldb://root:root@localhost/ouija2', echo=True)
+DBURL = urlparse.urlparse(os.environ["DATABASE_URL"])
```

Comment:
if we run this locally, what value do we need in the DATABASE_URL env?  I need more information, ideally an example one and how it is set in production (if this comes by default in heroku, etc.)